packages/marketplace/src/components/ui/developer-invite-member-modal/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './developer-invite-member-modal'
```

Comment:
Should we export default instead of export as object like this ?

Reply:
This is the export default syntax
It's equivalent to 
```
import DeveloperInviteModal from './developer-invite-member-modal'

export default DeveloperInviteModal
```

Reply:
@vuhuucuong ah ok